### ISSUE TITLE
Load layer: Global Administrative Boundaries V2

### ISSUE THREAD

**Issue by djtfmartin** [AtlasOfLivingAustralia/layers-service#6]
_migrated from:_ https://code.google.com/p/ala/issues/detail?id=68
_date:_ Thu Aug  8 05:37:04 2013
_author:_ moyesyside

---

Original Issue  - [https://code.google.com/p/alageospatialportal/issues/detail?id=971](https://code.google.com/p/alageospatialportal/issues/detail?id=971)

Project Member Reported by chris.fl...@gmail.com, May 8, 2012 
Organisation: Berkeley University
Contact: John Wiezorek
Link: [http://www.gdam.org](http://www.gdam.org)
Type: Contextual
 May 21, 2012 Project Member `#1` leebel...@gmail.com 
(No comment was entered for this change.)
 Labels: Type-Enhancement

**Comment by ansell** [comment]
PSMA Australia Limited provide much (all?) of this information for Australia. See issue #81 for the state there. All of the PSMA data is now loaded to spatial-test and waiting for the Spatial sprint next week to start to get it into production.

PSMA also provide their data under CC-BY, where GADM have a non-commercial license attached to their data, so we wouldn't want to preference GADM for Australian boundaries based on that.

**Comment by Tasilee** [comment]
Agreed. If we have PSMA suburb/area boundaries, they should be used in preference to GADM.
